LPS-141475 Migrate Data Set Display frontend to a dedicated app module

markocikos opened the pull request:
JIRA: https://issues.liferay.com/browse/LPS-141475

This is the first part of [rebuilding Data Set Display as a DXP app](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/1912931035/LPS-140691+-+Create+a+strategy+to+rebuild+Data+Set+Display+as+a+DXP+app).
- [x] data-set-web
- [ ] data-set-sample-web
- [ ] data-set-api
- [ ] data-set-taglib
- [ ] data-set-util

In the PR, we are:
- Extracting DSD frontend to a dedicated module. It consists of `DataSetDisplay` React component and a few exposed JS utilities.
- Reducing  `<clay:*data-set-display>` tags to thin wrappers around the component.
- Importing exposed utilities, removing the hardcoded paths.

To test, open any Data Set Display in DXP. They are on almost all pages in Commerce. @FabioDiegoMastrorilli please test the commerce mock environment. I did not know how to test this, so it is quite likely to contain a bug.

/cc @nhpatt @carloslancha @javierdearcos @izaera @jbalsas @dsanz

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 11 out of 11 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 25 out of 27 jobs passed in 1 hour 53 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0990df6c2831c0bb3dccb9450d48ec0deaaaaa35">0990df6c2831c0bb3dccb9450d48ec0deaaaaa35</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fd7c1fdd6785b11a50ceae5e5f821e13b0414861">fd7c1fdd6785b11a50ceae5e5f821e13b0414861</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2514">Acceptance Upstream DXP (master) #2514</a></p><details><summary><strong>ci:test:stable - 11 out of 11 jobs PASSED</strong></summary><details><summary><strong>11 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483220/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479282/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/236353/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/10826/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/504490/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/366608/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483221/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18425/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/472012/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18424/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483219/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 24 out of 27 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/1023/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2356/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2735/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/480587/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2733/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18426/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/504491/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483220/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/472013/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13645/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479282/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/236353/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/480588/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/10826/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/504490/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/236352/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/479281/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/366608/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2353/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/366609/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483221/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18425/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/472012/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2734/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/18424/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13644/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/483219/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1023/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at fd7c1fdd6785b11a50ceae5e5f821e13b0414861:</strong></summary><ol><li><div><h5><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2356/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></h5><div><h6>Job Results:</h6><p>3 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/2356//consoleText">AXIS_VARIABLE=0 #2356</a><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/2356//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanOSGiLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanOSGiLog</a><pre><code>junit.framework.AssertionFailedError: 
Portal log assert failure, OSGi log found: /opt/dev/projects/github/liferay-portal/bundles/osgi/state/1635528456169.log:
!SESSION 2021-10-29 17:28:02.089 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_121
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US

!ENTRY com.liferay.site.initializer.extender 4 0 2021-10-29 17:28:02.089
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: com.liferay.site.initializer.extender [663]
  Unresolved requirement: Import-Package: com.liferay.commerce.account.util; version="1.3.0"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1682)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1662)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1624)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1555)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)


	at com.lifer...</code></pre></div></li><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/2356//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
FrameworkEvent ERROR 
org.osgi.framework.BundleException: Could not resolve module: com.liferay.site.initializer.extender [663]
  Unresolved requirement: Import-Package: com.liferay.commerce.account.util; version="1.3.0"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1682)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1662)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1624)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1555)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.asser...</code></pre></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2735/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2396 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/2735//consoleText">AXIS_VARIABLE=1 #2735</a><ol><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/2735//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to find default image scaler
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/1023/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/1023/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/1023/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/1023/">test-portal-acceptance-pullrequest(master)#1023</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1023/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1609">liferay-frontend#1609</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1573963495">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#1609 - 2021-10-29[09:21:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/3062/">publish-testray-report#3062</a><br/></div></div>

izaera commented:
LNHTM :+1: (the server and build parts, I haven't reviewed .js files)

markocikos commented on modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/js/DataSetDisplayTag.js:
```diff
@@ -0,0 +1,80 @@
+	views,
+}) => {
+	return (
+		<DataSetDisplay
```
> We can just receive props and then pass {...props} on DataSetDisplay component.

Oh yeah, definitely :+1: 

> Also, does make sense adding PropTypes here? I think it could be useful for throwing errors in runtime(when receveing props)

No need after the `props` change :slightly_smiling_face:

markocikos commented:
Force-pushed. Changes:
- simplified `DataSetDisplayTag` based on https://github.com/liferay-frontend/liferay-portal/pull/1609#discussion_r739426024
- updated `@clayui/*` to 3.39.0
- fixed semver conflict

Pushing!

markocikos commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-source-format/935/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com//userContent/jobs/test-portal-source-format/builds/935/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/736/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/736/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fe4dba4a3ba06aa388de4cd1c9ff67e31b7d5791">fe4dba4a3ba06aa388de4cd1c9ff67e31b7d5791</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/markocikos/liferay-portal/tree/LPS-141475">LPS-141475</a><br/>Branch GIT ID: <a href="https://github.com/markocikos/liferay-portal/commit/53c4a5bff5f41d8eba7e380b77395b015f84fd6d">53c4a5bff5f41d8eba7e380b77395b015f84fd6d</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/935/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/935/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-source-format/935/">test-portal-source-format#935</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/935/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1609">liferay-frontend#1609</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1574929397">[master] ci:test:sf - markocikos &gt; liferay-frontend - PR#1609 - 2021-11-02[05:48:19]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/3054/">publish-testray-report#3054</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 11 out of 11 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 25 out of 27 jobs passed in 1 hour 39 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fe4dba4a3ba06aa388de4cd1c9ff67e31b7d5791">fe4dba4a3ba06aa388de4cd1c9ff67e31b7d5791</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/02af9eeedb35fe861a237716f7a420a40db96d2b">02af9eeedb35fe861a237716f7a420a40db96d2b</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2526">Acceptance Upstream DXP (master) #2526</a></p><details><summary><strong>ci:test:stable - 11 out of 11 jobs PASSED</strong></summary><details><summary><strong>11 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13719/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11116/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6992/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13720/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11764/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11765/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6995/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11766/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6985/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11762/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11115/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 24 out of 27 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/736/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/7002/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11117/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>24 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6986/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11112/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6994/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6989/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13719/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11767/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11114/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11116/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6992/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6990/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13720/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11764/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6988/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11763/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11765/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11113/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6991/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6995/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11766/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6985/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6987/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11762/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/6984/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11115/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/736/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 02af9eeedb35fe861a237716f7a420a40db96d2b:</strong></summary><ol><li><div><h5><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/7002/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></h5><div><h6>Job Results:</h6><p>3 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/7002//consoleText">AXIS_VARIABLE=0 #7002</a><ol><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/7002//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanOSGiLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanOSGiLog</a></div></li><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/7002//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11117/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2395 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/11117//consoleText">AXIS_VARIABLE=1 #11117</a><ol><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/11117//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to find default image scaler
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=4/11117//consoleText">AXIS_VARIABLE=4 #11117</a><ol><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=4/11117//testReport/com.liferay.batch.planner.service.test/BatchPlannerPlanServiceTest/testAddBatchPlannerPlan">com.liferay.batch.planner.service.test.BatchPlannerPlanServiceTest.testAddBatchPlannerPlan</a><pre><code>java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.fail(Assert.java:96)
	at com.liferay.batch.planner.service.test.BatchPlannerPlanServiceTest.testAddBatchPlannerPlan(BatchPlannerPlanServiceTest.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$1.evaluate(TestExecutorRunnable.java:237)
	at com.life...</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/736/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/736/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/736/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/736//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/109367
[Console](https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/736//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/736/">test-portal-acceptance-pullrequest(master)#736</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/736/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/1609">liferay-frontend#1609</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1574989774">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#1609 - 2021-11-02[05:48:14]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/2247/">publish-testray-report#2247</a><br/></div></div>
